adminSiteClient/DataInsightIndexPage.tsx

```diff
@@ -533,6 +542,20 @@ export function DataInsightIndexPage() {
                             }
                         />
                     )}
+                    {isCreateModalOpen && (
+                        <CreateDataInsightModal
+                            closeModal={() => setIsCreateModalOpen(false)}
+                            onFinish={(response) => {
```

Comment:
Can we refresh the index page also? It'd be nice to see the newly created DI in the list.

Ideally via another `GET` to `admin/api/dataInsights`